pkg/daemon/ceph/osd/daemon.go

```diff
@@ -34,10 +34,15 @@ import (
 	"github.com/rook/rook/pkg/operator/ceph/cluster/osd/config"
 	cephver "github.com/rook/rook/pkg/operator/ceph/version"
 	"github.com/rook/rook/pkg/operator/k8sutil"
+	"github.com/rook/rook/pkg/util/display"
 	"github.com/rook/rook/pkg/util/sys"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
 )
 
+const (
+	fiveGiBInBits uint64 = 42949672960 // this is 5GiB in Bits
```

Comment:
How about this for readability? If we're passing --bytes to lsblk I wouldn't expect bits.
```
fiveGiBInBytes uint64 = 5 * 1024 * 1024 * 1024
```